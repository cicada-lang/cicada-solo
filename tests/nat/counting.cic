n = zero
one = add1(n)

one

the(Nat, add1(add1(zero)))
