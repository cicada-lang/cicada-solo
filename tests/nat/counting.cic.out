1: Nat
2: Nat
